gtsfm/utils/ellipsoid.py

```diff
@@ -0,0 +1,163 @@
+"""Algorithms to fit Ellisoid geometry to 3D points and extract the rotation matrix necessary to align point cloud with
+x, y, and z axes. Used in React Three Fiber Visualization Tool.
+
+Authors: Adi Singh
+"""
+
+import numpy as np
+from numpy.linalg import eig, inv
```

Comment:
For readability, we don’t import functions, could you please use np.linalg.inv()?